geode-core/src/test/java/org/apache/geode/internal/cache/FixedPRSinglehopDUnitTest.java

```diff
@@ -293,18 +294,23 @@ public void test_FPAmetadataFetch() {
       putIntoPartitionedRegionsThreeQs();
 
       getFromPartitionedRegionsFor3Qs();
-      Wait.pause(2000);
+      // Server 1 is actually primary for both Q1 and Q2, since there is no FPA server with
+      // primary set to true.
+      Awaitility.await().atMost(15, TimeUnit.SECONDS).until(
+          () -> (server1.invoke(() -> FixedPRSinglehopDUnitTest.primaryBucketsOnServer()) == 6)
```

Comment:
I think you can simplify the lambda expression passed to `server1.invoke()` by using a method reference instead:
```
() -> (server1.invoke(FixedPRSinglehopDUnitTest::primaryBucketsOnServer) == 6)
```

If `primaryBucketsOnServer` was not a static method, it would look even shorter:
```
() -> (server1.invoke(this::primaryBucketsOnServer) == 6)
```